jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ImageToJsonTranslator.java

```diff
@@ -59,26 +60,34 @@
   @VisibleForTesting
   @Nullable
   static Map<String, Map<?, ?>> portListToMap(@Nullable List<Port> exposedPorts) {
-    if (exposedPorts == null) {
-      return null;
-    }
+    return listToMap(exposedPorts, port -> port.getPort() + "/" + port.getProtocol());
+  }
 
-    return listToMap(
-        exposedPorts
-            .stream()
-            .map(port -> port.getPort() + "/" + port.getProtocol())
-            .collect(Collectors.toList()));
+  /**
+   * Converts a list of {@link AbsoluteUnixPath}s to the corresponding container config format for
+   * volumes (e.g. {@code AbsoluteUnixPath().get("/var/log/my-app-logs")} -> {@code
+   * {"/var/log/my-app-logs":{}}}).
+   *
+   * @param volumes the list of {@link AbsoluteUnixPath}s to translate, or {@code null}
+   * @return a sorted map with the string representation of the ports as keys and empty maps as
+   *     values, or {@code null} if {@code exposedPorts} is {@code null}
+   */
+  @VisibleForTesting
+  @Nullable
+  static Map<String, Map<?, ?>> volumesListToMap(@Nullable List<AbsoluteUnixPath> volumes) {
+    return listToMap(volumes, AbsoluteUnixPath::toString);
   }
 
   @Nullable
-  static Map<String, Map<?, ?>> listToMap(@Nullable List<String> list) {
+  static <E> Map<String, Map<?, ?>> listToMap(
```

Comment:
Might want to make a note that this method exists because the JSON form ["is a direct JSON serialization of the Go type map[string]struct{} and is represented in JSON as an object mapping its keys to an empty object"](https://github.com/opencontainers/image-spec/blob/master/config.md).

Reply:
Added this on the javadoc.